modules/appeals_api/spec/support/schemas/legacy_appeal.json

```diff
@@ -0,0 +1,46 @@
+{
+  "properties": {
+    "type": {
+      "type": "string",
+      "example": "legacyAppeal"
+    },
+    "id": {
+      "type": "string",
+      "description": "ID from VACOLS (Veteran Appeals Control and Locator Service)",
+      "example": "3085659"
+    },
+    "attributes": {
+      "type": "object",
+      "properties": {
+        "issues": {
+          "type": "array",
+          "description": "Issues on the Appeal.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "summary": {
+                "type": "string",
+                "description": "Summary of a single Issue.",
+                "example": "Service connection, hearing loss"
+              }
+            }
+          }
+        }
+      },
+      "veteranFullName": {
+        "type": "string",
+        "example": "Junior L Fritsch"
+      },
+      "decisionDate": {
+        "type": "string",
+        "description": "Date of the Appeal's original decision.",
+        "example": "2018-09-28T00:00:00.000Z"
+      },
+      "latestSocSsocDate": {
+        "type": "string",
+        "description": "Date of the Appeal's most recent SOC/SSOC (Statement of Case/Supplemental Statement of Case)",
+        "example": "2018-12-29T00:00:00.000Z"
+      }
```

Comment:
I think these keys should be moved 'up' to be inside the `attributes->properties`... Right now they aren't showing up in the schema like they should.

Reply:
Oy. Yes. Thanks for catching that